### PR TITLE
Synchronize kubeflow model registry manifests v0.2.5 alpha

### PR DESCRIPTION
**Pull request by juliusvonkohout**
@lampajr @rimolive Did you change the manifest structure? because i do not see any real changes.

### CODE REVIEW

**Comment by tarilabs** [comment]
as mentioned [in slack](https://cloud-native.slack.com/archives/C073W572LA2/p1726313759857289?thread_ts=1726313411.856479&cid=C073W572LA2), I confirm no structure change for MR Manifest.

File: README.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ This repo periodically syncs all official Kubeflow components from their respect
 | KServe Models Web App | contrib/kserve/models-web-app | [0.13.0](https://github.com/kserve/models-web-app/tree/0.13.0/config) |
 | Kubeflow Pipelines | apps/pipeline/upstream | [2.3.0](https://github.com/kubeflow/pipelines/tree/2.3.0/manifests/kustomize) |
 | Kubeflow Tekton Pipelines | apps/kfp-tekton/upstream | [2.0.5](https://github.com/kubeflow/kfp-tekton/tree/2.0.5/manifests/kustomize) |
-| Kubeflow Model Registry | apps/model-registry/upstream | [v0.2.1-alpha](https://github.com/kubeflow/model-registry/tree/v0.2.1-alpha/manifests/kustomize) |
+| Kubeflow Model Registry | apps/model-registry/upstream | [v0.2.5-alpha](https://github.com/kubeflow/model-registry/tree/v0.2.5-alpha/manifests/kustomize) |
```
Comment (tarilabs) [inline README.md:61]:
```suggestion
| Kubeflow Model Registry | apps/model-registry/upstream | [v0.2.6-alpha](https://github.com/kubeflow/model-registry/tree/v0.2.5-alpha/manifests/kustomize) |
```

File: hack/synchronize-model-registry-manifests.sh [hunk 2]
```diff
@@ -15,7 +15,7 @@
 set -euxo pipefail
 IFS=$'\n\t'
 
-COMMIT="v0.2.1-alpha" # You can use tags as well
+COMMIT="v0.2.5-alpha" # You can use tags as well
```
Comment (tarilabs) [inline hack/synchronize-model-registry-manifests.sh:18]:
```suggestion
COMMIT="v0.2.6-alpha" # You can use tags as well
```

**Review by tarilabs — COMMENTED** [review]
thank you @juliusvonkohout

**Comment by juliusvonkohout** [comment]
Then pease /lgtm or create another PR if you do not like this one.

**Comment by tarilabs** [comment]
I've never said I didn't like this one, I proposed while it was open to simply update with the just fresh tag. 

/lgtm

We can do another update in a follow-up PR if that is preferred 👍 

Thank you

**Review by tarilabs — APPROVED** [review]

**Pull request merged**